includes/blocks/example-block/index.css

```diff
@@ -0,0 +1,7 @@
+/**
+ * Example Block
```

Comment:
@ryanwelcher 

Continuation on the css discussion in a previous comment:
Should we be concerned about enqueuing / loading a css file for each block in the admin?

Reply:
There is an entry for that in in the `block.json` file as well. Perhaps I need to provide more robust examples here.